### ISSUE TITLE
Lightning tools adjustments

### ISSUE THREAD

**Issue by fallenthereaper** [AlexModGuy/Ice-and-Fire-Suggestions#1034]
Instead of summoning lightning bolt(which is literaly suicide) could maybe make it summon some small lightning strikes(similar to lightning dragon breath but smaller) surrounding the target and giving damage, electrocuted to death because summoning a lightning bolt doesn't make sense imo, fire burns the entity, ice freezes why would lightning summon a lightning bolt which can literally kill the player itself.

Here's some examples:
![image](https://user-images.githubusercontent.com/79212718/108353493-1c84a480-71e0-11eb-94fc-5fa5b81e50f0.png)
![image](https://user-images.githubusercontent.com/79212718/108353546-31613800-71e0-11eb-8289-015e0d9bcf5b.png)

**Comment by 1Melone** [comment]
Same idea

**Comment by 1Melone** [comment]
Nice photos!

**Comment by NeutronForge** [comment]
Yes. We need this adjustment and also why can you hear the lightning on a server along the whole map and thru dimensions? Also lightning burns the drops, which is in my eyes even more annoying
